pkg/splunk/enterprise/util_test.go

```diff
@@ -1854,3 +1854,209 @@ func TestRemoveStaleEntriesFromAuxPhaseInfo(t *testing.T) {
 		t.Errorf("removeStaleEntriesFromAuxPhaseInfo should have cleared the last 2 entries from AuxPhaseInfo")
 	}
 }
+
+func TestMigrateAfwStatus(t *testing.T) {
+	cr := &enterpriseApi.Standalone{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "Standalone",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "stack1",
+			Namespace: "test",
+		},
+	}
+
+	statefulSetName := "splunk-stack1-standalone"
+	var replicas int32 = 4
+
+	sts := &appsv1.StatefulSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      statefulSetName,
+			Namespace: "test",
+		},
+		Spec: appsv1.StatefulSetSpec{
+			Replicas: &replicas,
+		},
+	}
+
+	client := spltest.NewMockClient()
+	_, err := splctrl.ApplyStatefulSet(client, sts)
```

Comment:
Do we actually need to apply statefulset or we can just add it to the mock client?

Reply:
Yeah that's why I said if you just add it to mock client, the get client won't fail :)